infra/scripts/misspell.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+out=$(misspell -error $(find . -mindepth 1 -maxdepth 1 -type d -not -name vendor) | grep -v -e lib/config/dynamic/admiral/client | grep -v -e lib/config/dynamic/admiral/models | grep -v -e lib/config/dynamic/admiral/swagger.json)
```

Comment:
same as above - these seem like files we should be either:
1. getting from the admiral repo, or 
2. generating per build from swagger (from admirals repo) and therefore not in vmware/vic SCM.

If (2) is actually the case, perhaps we should look at having goimport/lint/misspell only run against SCM managed files.